components/GlobalStyle/GlobalStyle.ts

```diff
@@ -223,6 +228,7 @@ a:not([class]) {
     --header-xs: ${headerXs};
     --text-lg: ${textLg};
     --text-md: ${textMd};
+    --text-md18: ${textMd18};
```

Comment:
Seems like we need to re-sort the fonts. Can you please add the missing ones and rename them so that we have either all numbers like `text-md-18` or add more levels so like `text-xxs`, `text-xs` etc. Makes it easier to stay consistent.